tests/test_migrate_coverage.py

```
@@ -0,0 +1,996 @@
+        # Should return the base path of the most recent group
+        assert result == new_db
+
+    def test_get_most_recent_database_empty_group(self, tmp_path):
+        """Test when db_groups is empty after filtering."""
+        # Create candidate that will be filtered out (doesn't exist)
+        non_existent = tmp_path / "nonexistent.sqlite"
+        candidates = [non_existent]
+
+        with mock.patch("mmrelay.migrate._get_most_recent_database") as mock_func:
+            # Mock to return empty list after filtering
+            def side_effect(cands):
```

Ruff (0.14.14)

[warning] Unused function argument: `cands`

(ARG001)